modules/wri_subpage/templates/wri-subpage-columns.html.twig

```diff
@@ -45,8 +45,8 @@
     {% endif %}
     {# end if content.hero #}
 
-    {% if content.category or content.share %}
-      {% if (content.hero.field_toc is not empty) and (content.hero.field_toc['#value'] != 'microsite') %}
```

Comment:
Keeping `content.hero.field_toc['#value'] != 'microsite'` should still be needed, so we don't get the mobile menu on pages like this one:
http://web.wriflagship.localhost/paying-for-paris/how-much-will-it-cost

The Cities page is a "microsite" content type but doesn't have the `field_toc` value of `microsite`.